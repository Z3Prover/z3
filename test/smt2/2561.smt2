(declare-const v Bool)
(assert (= v (= 0.0 (/ 1.0 0.0))))
(check-sat)
(get-model)
(eval v)
(exit)

(declare-const v Bool)
(declare-const w Bool)
(declare-const r Real)
(declare-const x Real)
(declare-const y Real)

(assert (> x (/ 1.0 0.0)))
(assert (= y (/ 2.0 0.0)))
(assert (= y (/ 3.0 0.0)))
(assert (= v (= 0.0 (/ 1.0 0.0))))
(assert (= w (= 0.0 (/ 1.0 r))))
(check-sat)
(eval v)
(eval w)
(eval (/ 1.0 0.0))
(eval x)
(get-model)