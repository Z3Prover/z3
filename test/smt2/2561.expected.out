sat
(model 
  (define-fun v () Bool
    true)
)
true
